src/Command/CleanUpCommand.php

```diff
@@ -51,15 +52,19 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $query = $this->coverRepository->getIsNotUploadedQuery($limit);
         /** @var Cover $cover */
         foreach ($query->toIterable() as $cover) {
-            if ($this->coverStoreService->exists($cover->getMaterial()->getIsIdentifier())) {
-                $this->coverStoreService->removeLocalFile($cover);
+            try {
+                if ($this->coverStoreService->exists($cover->getMaterial()->getIsIdentifier())) {
+                    $this->coverStoreService->removeLocalFile($cover);
```

Comment:
I know this is only a refactor. But I really think it was the wrong decision to not keep a local copy of the file. A CDN is not file storage.  As the need for this PR clearly demonstrates.

Would prefer we remove this command.